.github/workflows/ci-pull.yml

```diff
@@ -18,10 +18,36 @@ jobs:
   build:
     runs-on: ubuntu-latest
 
-    #services: TODO add couchdb container for testing
+    services: 
+      cloudant:
+        #Update Readme to match
+        image: couchdb:latest
+        env:
+            COUCHDB_USER: admin
+            COUCHDB_PASSWORD: password
+        # Set health checks to wait until couchdb has started
+        options: >-
+          --health-cmd "curl -f http://localhost:5984/_up || exit 1"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5984:5984
         
 
     steps:
+    # Start CouchDB and initialize required databases
+    - name: Wait for CouchDB to be ready
+      run: |
+        until curl -s -u admin:password http://localhost:5984/_up; do
+          echo "Waiting for CouchDB..."
+          sleep 5
+        done
+    - name: Create system databases
```

Comment:
It's strange to me that if these are necessary they aren't included in the container image, are you sure these need to be created?